discovery-provider/src/tasks/index_trending.py

```diff
@@ -82,10 +85,11 @@ def index_trending(self, db, redis):
         # Make sure to cache empty genre
         genres.append(None)
 
+        strategy = trending_strategy_factory.get_strategy(TrendingType.TRACKS)
```

Comment:
I think we need another for loop here

for strategy in strategies:
  for genre in genres:
     ... 


didn't we have a bit before where we put the strategy into the cache key?

Reply:
the trending strategy/version is still there as part of the cache key, and it takes the default version. whether we need the loop for strategies depends on whether we want to cache non default strategies

Reply:
I think we definitely do -- it's pretty slow to run the queries -- we should cache everything, but because of that, we probably want to keep # of experiments relatively low